domain/src/main/scala/com/all4journey/domain/Tables.scala

```diff
@@ -147,34 +145,69 @@ trait Tables {
   /** Collection-like TableQuery object for table SchemaVersion */
   lazy val SchemaVersion = new TableQuery(tag => new SchemaVersion(tag))
 
+  /** Entity class storing rows of table Token
+    *  @param id Database column ID SqlType(VARCHAR), PrimaryKey, Length(50,true)
+    *  @param userId Database column USER_ID SqlType(VARCHAR), Length(50,true)
+    *  @param token Database column TOKEN SqlType(VARCHAR), Length(50,true) */
+  case class TokenRow(id: String, userId: String, token: String)
+  /** GetResult implicit for fetching TokenRow objects using plain SQL queries */
+  implicit def GetResultTokenRow(implicit e0: GR[String]): GR[TokenRow] = GR{
+    prs => import prs._
+      TokenRow.tupled((<<[String], <<[String], <<[String]))
+  }
+  /** Table description of table TOKEN. Objects of this class serve as prototypes for rows in queries. */
+  class Token(_tableTag: Tag) extends Table[TokenRow](_tableTag, "TOKEN") {
+    def * = (id, userId, token) <> (TokenRow.tupled, TokenRow.unapply)
+    /** Maps whole row to an option. Useful for outer joins. */
+    def ? = (Rep.Some(id), Rep.Some(userId), Rep.Some(token)).shaped.<>({r=>import r._; _1.map(_=> TokenRow.tupled((_1.get, _2.get, _3.get)))}, (_:Any) =>  throw new Exception("Inserting into ? projection not supported."))
+
+    /** Database column ID SqlType(VARCHAR), PrimaryKey, Length(50,true) */
+    val id: Rep[String] = column[String]("ID", O.PrimaryKey, O.Length(50,varying=true))
+    /** Database column USER_ID SqlType(VARCHAR), Length(50,true) */
+    val userId: Rep[String] = column[String]("USER_ID", O.Length(50,varying=true))
+    /** Database column TOKEN SqlType(VARCHAR), Length(50,true) */
+    val token: Rep[String] = column[String]("TOKEN", O.Length(50,varying=true))
+
+    /** Foreign key referencing User (database name TOKEN_ibfk_1) */
+    lazy val userFk = foreignKey("TOKEN_ibfk_1", userId, User)(r => r.id, onUpdate=ForeignKeyAction.NoAction, onDelete=ForeignKeyAction.Cascade)
+  }
+  /** Collection-like TableQuery object for table Token */
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Field name 'Token' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/rjkernick/all4journey/file/2445603256/issues/source?bid=3246460&fileBranchId=3246461#l175)